src/node/utils/Settings.js

```diff
@@ -430,6 +430,11 @@ exports.importMaxFileSize = 50 * 1024 * 1024;
  */
 exports.enableAdminUITests = false;
 
+/*
+ * Enable case-insensitive pad names.
+ */
+exports.caseInsensitivePadNames = false;
```

Comment:
I wonder if renaming this to something like `lowerCasePadNames` would be better. Technically it's not case-insensitive—if pad `FooBar` exists and the user visits `Foobar` then they'll create a new `foobar` pad, not visit the existing `FooBar` pad.

Reply:
I also thought about that, I will update it accordingly!

Reply:
Actually, I wonder if `lowerCasePadIds` is better than `lowerCasePadNames`. I think we use "pad ID" more than "pad name" in both user-visible stuff and in source code.